src/main/data/facilities.json

```diff
@@ -1516,6 +1516,16 @@
       "FAC"
     ]
   },
+  {
+    "code": "CSM",
+    "contact": {
+      "address": "London, United Kingdom",
```

Comment:
```suggestion
```

Let's remove the incomplete address.

Reply:
I think it should remain. I think it gives the region of interest to folks. Granted it's not the mailing address I think there is value to City/State.

Reply:
It breaks automation that assumes that a complete mailing address is present. The submitter should submit a complete address.

Reply:
I have added a full address...